packages/grid/_modules_/grid/models/gridSlotsComponent.ts

```diff
@@ -8,59 +8,73 @@ import { GridIconSlotsComponent } from './gridIconSlotsComponent';
 export interface GridSlotsComponent extends GridIconSlotsComponent {
   /**
    * The custom Checkbox component used in the grid for both header and cells.
+   * @default Checkbox
    */
   Checkbox: React.JSXElementConstructor<any>;
   /**
    * Column menu component rendered by clicking on the 3 dots "kebab" icon in column headers.
+   * @default GridColumnMenu
    */
   ColumnMenu: React.JSXElementConstructor<any>;
   /**
    * Error overlay component rendered above the grid when an error is caught.
+   * @default ErrorOverlay
    */
   ErrorOverlay: React.JSXElementConstructor<any>;
   /**
    * Footer component rendered at the bottom of the grid viewport.
+   * @default GridFooter
    */
   Footer: React.JSXElementConstructor<any>;
   /**
    * Header component rendered above the grid column header bar.
-   * Prefer using the `Toolbar` slot. You should never need to use this slot. TODO remove.
+   * Prefer using the `Toolbar` slot. You should never need to use this slot.
+   * @default GridHeader
    */
-  Header: React.JSXElementConstructor<any>;
+  Header: React.JSXElementConstructor<any>; // TODO remove.
   /**
    * Toolbar component rendered inside the Header component.
+   * @default null
    */
   Toolbar: React.JSXElementConstructor<any> | null;
   /**
    * PreferencesPanel component rendered inside the Header component.
+   * @default GridPreferencesPanel
    */
   PreferencesPanel: React.JSXElementConstructor<any>;
   /**
    * Loading overlay component rendered when the grid is in a loading state.
+   * @default GridLoadingOverlay
    */
   LoadingOverlay: React.JSXElementConstructor<any>;
   /**
    * No results overlay component rendered when the grid has no results after filtering.
+   * @default GridNoResultsOverlay
    */
   NoResultsOverlay: React.JSXElementConstructor<any>;
   /**
    * No rows overlay component rendered when the grid has no rows.
+   * @default GridNoRowsOverlay
    */
   NoRowsOverlay: React.JSXElementConstructor<any>;
   /**
    * Pagination component rendered in the grid footer by default.
+   * @default Pagination
    */
-  Pagination: React.JSXElementConstructor<any> | null;
+  Pagination: React.JSXElementConstructor<any>;
```

Comment:
If you remove the `null` here, you can remove the `rootProps.components.Pagination &&` in `GridFooter`

Reply:
Reverted.